tox.ini

```diff
@@ -99,6 +111,7 @@ extras =
 
 commands =
     devdeps: pip install -U --pre --only-binary :all: -i https://pypi.anaconda.org/scipy-wheels-nightly/simple numpy
+    devdeps: pip install -U --pre --only-binary :all: -i https://pypi.anaconda.org/scipy-wheels-nightly/simple scipy
```

Comment:
https://pypi.anaconda.org/scipy-wheels-nightly also has nightly wheels for matplotlib, could we use that instead of installing from git ?

Reply:
Sure, why not. Might as well be consistent.

Reply:
OK, done. Thanks for the suggestion!